tests/test_action.py

```diff
@@ -353,6 +353,21 @@ def error_sample(): return object()
         got = my_action.execute()
         assert isinstance(got, TaskError)
 
+    def test_error_taskfail(self):
+        # should get the same exception as was returned from the
+        # user's function
+        def error_sample(): return TaskFailed("too bad")
+        ye_olde_action = action.PythonAction(error_sample)
```

Comment:
ok, got it. :+1:  http://en.wikipedia.org/wiki/Ye_Olde

I was thinking it was some kind of acronym... you can leave it this way :)